test/test-v8debugapi.js

```diff
@@ -700,6 +700,31 @@ describe('v8debugapi', function() {
         process.nextTick(function() {foo();});
       });
     });
+
+    it('should not silence errors thrown in the wait callback', function(done) {
+      var message = 'This exception should not be silenced';
+      // Remove the mocha listener.
+      var originalListener = process.listeners('uncaughtException').pop();
```

Comment:
Won't `pop` and `removeListener` attempt the same action? Also we may want to assert that there is only a single listener at this point to ensure we're always grabbing the mocha listener or that there aren't more attached by mocha we need to be aware of.